src/03_modules.py

```diff
@@ -10,22 +10,29 @@
 
 # Print out the command line arguments in sys.argv, one per line:
 # YOUR CODE HERE
+# going to look more into this one 
```

Comment:
Great! sys.argv is a python list which stores the command line args passed in your teminal. How do we print out the each element in the list?